src/main/java/org/fineract/messagegateway/sms/api/SmsApiResource.java

```diff
@@ -50,7 +55,18 @@ public SmsApiResource(final SMSMessageService smsMessageService) {
     public ResponseEntity<Void> sendShortMessages(@RequestHeader(MessageGatewayConstants.TENANT_IDENTIFIER_HEADER) final String tenantId,
     		@RequestHeader(MessageGatewayConstants.TENANT_APPKEY_HEADER) final String appKey, 
     		@RequestBody final List<SMSMessage> payload) {
-    	this.smsMessageService.sendShortMessage(tenantId, appKey, payload);
+        
+    	WirepickSMS sms = new WirepickSMS() ;
+         for(SMSMessage sMessage : payload) {
+            WpkClientConfig config = new WpkClientConfig(ConstantValues.SMS_CLIENT_ID, ConstantValues.SMS_CLIENT_PWD, 
+                    ConstantValues.SMS_CLIENT_AFFILIATE,smessage.getMobileNumber() , smessage.getMessage(), ConstantValues.SMS_CLIENT_SENDER_ID, ConstantValues.SMS_TAG) ; 
+            try {
+                MsgStatus msgStatus =  sms.SendPOSTSMS(config) ;
+                System.out.println(msgStatus.getMessageId());
+            } catch (Exception e) {
+            }
```

Comment:
In the exception, please print something so that the admin knows what is the error.

Reply:
done

Reply:
But print this through `System.out.println()`